### ISSUE TITLE
ellispsis in simInitAndSpades is not being "respected

### ISSUE THREAD

**Issue by CeresBarros** [PredictiveEcology/SpaDES.core#172]

**Comment by CeresBarros** [comment]
I'm trying to pass `.plots` to `spades` via `simInitAndSpades` but the following code ignores `...` arguments:
```
  lsAllNames <- ls(all.names = TRUE)
  lsAllNames <- lsAllNames[lsAllNames != "..."]
  objsAll <- mget(lsAllNames, envir = environment())
  ...
  spadesFormals <- formalArgs(spades)[formalArgs(spades) %in% 
    names(objsAll)]  
```

**Comment by CeresBarros** [comment]
a temporary fix suggested in #169

**Comment by CeresBarros** [comment]
fixed in #169. now merged into develpment
